@kiva/kv-components/src/vue/KvCardFrame.vue

```diff
@@ -2,10 +2,11 @@
 	<kv-theme-provider
 		:theme="themeStyle"
 		class="kv-card-frame-theme-provider"
+		:class="radiusClass"
```

Comment:
This ensures the theme provider div matches the content area preventing odd styles from custom classes applied in different scenarios.